.github/workflows/publish_to_maven_local.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       fail-fast: false
       matrix:
         provider: [
-          cloudflare, slack, github, random, gcp, google-native, aws, aws-native, azure, azure-native, kubernetes,
-          nomad, docker, gitlab, digitalocean, alicloud
+          cloudflare4, cloudflare5, slack0, github5, random4, gcp6, google-native0, aws5, aws-native0, azure5,
+          azure-native1, kubernetes3, nomad0, docker3, docker4, gitlab4, gitlab5, gitlab6, digitalocean4, alicloud3
```

Comment:
These numbers might be surprising for an unfamiliar reader. Have you tried doing it this way?

```yaml
provider: 
  - name: cloudflare
     majorVersion: 4
  - name: cloudflare
     majorVersion: 5
#  ...
```

I don't know if the above will work, but using `include` might be an option here

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations

Reply:
I created an issue for that: https://github.com/VirtuslabRnD/pulumi-kotlin/issues/294